test/integration/backward_compatible/topic/ReliableTopicOnClusterRestart.js

```diff
@@ -29,6 +29,10 @@ describe('ReliableTopicOnClusterRestartTest', function () {
     let client1;
     let client2;
 
+    before(function () {
+        markClientVersionAtLeast(this, '4.0.2');
```

Comment:
Maybe we can add the link of the PR here as a comment to make the life of the future-readers easier